src/Compilers/CSharp/Test/Symbol/Symbols/SymbolEqualityTests.cs

```diff
@@ -41,6 +42,11 @@ void M()
             var nonNullPlus = (IMethodSymbol)model.GetSymbolInfo(invocations[0]).Symbol;
             var nullPlus = (IMethodSymbol)model.GetSymbolInfo(invocations[1]).Symbol;
 
+            Assert.True(nonNullPlus.ImplementationAttributes == MethodImplAttributes.IL);
+            Assert.True(nonNullPlus.ImplementationAttributes == MethodImplAttributes.Managed);
+            Assert.True(nullPlus.ImplementationAttributes == MethodImplAttributes.IL);
+            Assert.True(nullPlus.ImplementationAttributes == MethodImplAttributes.Managed);
```

Comment:
@jcouv Before I attempt to write more tests, I want to confirm that I'm doing it the correct way. I feel that I'm missing something. Both IL and Managed are defined in the enum as 0, which confused me a little bit.

Reply:
If it's a managed method, it's implemented in MSIL. It's certainly not the greatest thing to have 2 labels, but it's out of our control :). No need to test equality to both. Also, please use `Assert.Equals`, not `Assert.True(==)`.

Reply:
@333fred Thanks. I've updated it to use `Assert.Equal`. Do you want me to keep updating tests in this file? or creating a brand new test file for testing this API?

Reply:
This file is fine.